gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Dialog.java

```diff
@@ -128,8 +128,8 @@ public Table getContentTable () {
 		return contentTable;
 	}
 
-	public Table getButtonTable () {
-		return buttonTable;
+	public Table getButtonsTable () {
```

Comment:
To avoid confusion with old method name. That can be changed though, if you think it would be better to leave old name.